### ISSUE TITLE
[BUG] 4.2.0 adds support for FCM v1...but no way to call SendFcmv1NativeNotificationAsync since it's missing from the INotificationHubClient interface

### ISSUE THREAD

**Issue by awaldow** [Azure/azure-notificationhubs-dotnet#307]
**Describe the bug**
Seems like Send... support for FCMv1 is missing from the interface - it has endpoints defined in the client impl itself but not the INotificationHubClient

https://github.com/Azure/azure-notificationhubs-dotnet/blob/main/src/Microsoft.Azure.NotificationHubs/INotificationHubClient.cs#L1408 shows the interface, absent the methods.

https://github.com/Azure/azure-notificationhubs-dotnet/blob/main/src/Microsoft.Azure.NotificationHubs/NotificationHubClient.cs#L639 shows the client having the desired methods.

***Exception or Stack Trace***

**To Reproduce**

1. Install 4.2.0
2. Notice you can't call SendFcmV1NativeNotificationAsync 

***Code Snippet***

**Expected behavior**
The underlying client impl methods are exposed on the interface.

**Screenshots**

**Setup (please complete the following information):**
 - OS: Windows 10
 - IDE : Visual Studio 2022,VS Code,
 - 4.2.0

**Additional context**

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by OculiViridi** [comment]
Is this issue solved?

**Comment by awaldow** [comment]
No, you can get around it by passing an `FcmV1Notification` to `SendNotificationAsync` though. There's just no first party method like the other platforms yet as far as I can see.

**Comment by Buyani** [comment]
How should a payload for "SendFcmV1NativeNotificationAsync" look like to cross platform  or native android or iOS?

**Comment by awaldow** [comment]
You would probably want to refer to the [Firebase docs](https://firebase.google.com/docs/cloud-messaging/migrate-v1#update-the-payload-of-send-requests) on payload formatting for FCMv1, since there were some changes in the shape of the message required.
